pkg/apis/openebs.io/v1alpha1/register.go

```diff
@@ -4,8 +4,6 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
-
-	"github.com/openebs/maya/pkg/apis/openebs.io"
```

Comment:
You should not remove this. It is being used in line number 12

Reply:
Sorry, I didn't mean to remove it, did that by accident probably. Fixing it right away.